developer/openshift/gitops/argocd/pipeline-service-o11y/kustomization.yaml

```diff
@@ -4,3 +4,4 @@ kind: Kustomization
 resources:
   - appstudio-prometheus
   - appstudio-grafana
+  - https://github.com/redhat-appstudio/infra-deployments/components/pipeline-service/base/servicemonitors?ref=main
```

Comment:
we (as in rhtap) have been moving away from defining component specific monitors in infra-deployments @sayan-biswas 

please follow the pattern I employed in https://github.com/openshift-pipelines/pipeline-service/pull/685 and https://github.com/redhat-appstudio/infra-deployments/pull/2014 and move the servicemonitor for results to this repo.

Reply:
Sure. I have created a separate PR #710 for this. I'll remove the service monitor changes from this PR.

Reply:
Yes. I removed everything else other than the dashboard.